.circleci/config.yml

```diff
@@ -108,6 +108,52 @@ jobs:
             - ./.pytype
           key: v1-pytype
 
+  test:
+      docker:
+          - image: circleci/python:3.7.4-node
+            environment:
+                APP_CONFIG: config.TestingConfig
+                SECRET_KEY: super secret key
+                PATH: $PATH:${HOME}/google-cloud-sdk/bin
+                DATABASE_URL: postgresql://test@localhost:5433/test
+                TEST_DATABASE_URL: postgresql://test@localhost:5433/test
```

Comment:
Don't know where the env is coming from. Match dredd job

Reply:
env was set in travis file, I copied them